bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2050,6 +2059,55 @@ protected void readLedgerIndexEntries(long ledgerId) throws IOException {
     }
 
     /**
+     * Read ledger index entries
+     * 
+     * @param ledgerId
+     * @return
+     * @throws IOException if failed to init KeyValueStorageRocksDB
+     */
+	protected int readLedgerIndexEntriesFromDbLedgerStorage(long ledgerId) throws IOException {
+
+		ServerConfiguration conf = new ServerConfiguration(bkConf);
+		LedgerDirsManager ledgerDirsManager = new LedgerDirsManager(bkConf, bkConf.getLedgerDirs());
+		String ledgersPath = ledgerDirsManager.getAllLedgerDirs().get(0).toString();
+
+		EntryLocationIndex entryLocationIndex = null;
+
+		try {
+			entryLocationIndex = new EntryLocationIndex(conf,
+					(path, dbConfigType, conf1) -> new KeyValueStorageRocksDB(path, DbConfigType.Small, conf1, true),
```

Comment:
This will still fail if the bookie process is running, because we try to open RocksDB in write mode